src/main/webapp/WEB-INF/views/session/login.jsp

```diff
@@ -301,11 +362,12 @@
 									<span class="checkSet"><input type="checkbox" id="saveID" /><label for="saveID">SAVE ID</label></span>
 									<strong><a class="btnRegist" id="btnRegist">SignUp</a></strong>
 								</span>
+								<span class="options">
+									<strong><a id="btnResetPwd">Reset Password</a></strong>
+								</span>
```

Comment:
Please move this code into the span tag where class name is "joinGo"(line 361-364).

Reply:
<img width="441" alt="image" src="https://user-images.githubusercontent.com/71131071/219027528-7f45ee26-58df-4d71-ba16-7a7771c9dabc.png">
Changed, thanks for the review!